### ISSUE TITLE
Comparator model 

### ISSUE THREAD

**Issue by sen-pai** [sen-pai/order-learning#4]
- [x] Simple CNN Base
- [x] Comparator Output (softmax)
- [x] Training Script

**Comment by sen-pai** [comment]
need to update README, with experiment details
